src/modules/app/fetchData.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import type { Dispatch } from 'redux-first-router'
+import type { Dispatch } from 'redux-first-router/dist/flow-types'
```

Comment:
this should be
```suggestion
import type { Dispatch } from 'redux'
```

(and the usage below should be `Dispatch<*>`

Reply:
Thanks :+1: